app/assets/v2/css/theme/dark/profile.css

```diff
@@ -193,6 +197,7 @@ a:hover .avatar_edit .avatar_edit_text {
 
 .profile-header__stats {
   border: none;
+  background: #000 !important;
```

Comment:
do we need to have the whole profile.css duplicated twice (once per theme)?  seems, in the spirit of DRY, like there should be a base profile.css, and then only the color augmentations should be listed in the light.css or dark.css

Reply:
Hmmmm, I think you are right. I'll update the affect areas